### ISSUE TITLE
Dear author, congrats for your work! will you share the video file of the interactions?

### ISSUE THREAD

**Issue by WenjiaWang0312** [jnnan/trumans_utils#4]

**Comment by jnnan** [comment]
Are you asking for the videos in the [project page](https://jnnan.github.io/trumans/)?

**Comment by WenjiaWang0312** [comment]
No, I mean the rendered RGBs corresponding to the HSI motions and environmenrts

**Comment by jnnan** [comment]
The 6 FPS videos and smplx parameters are now [released](https://drive.google.com/drive/folders/1axXjugOzxlWD_kJsY6v-cn5Q7k5V1YfL?usp=sharing)
